app.rb

```diff
@@ -0,0 +1,25 @@
+require 'sinatra/base'
+require './lib/chitter'
+
+class ChitterManager < Sinatra::Base
+  enable :sessions
+
+  get '/chitter' do
+    p ENV
+    @chitter = Chitter.all
+    erb :new
+  end
+
+  post '/chitter' do
+    message = params['message']
+    connection = PG.connect(dbname: 'chitter_manager_test')
```

Comment:
It looks like this is always pointing to your test environment? This could be configured depending on the environment variable in a similar way to your `self.all` method in your `Chitter` object